socorro/external/es/super_search_fields.py

```diff
@@ -539,8 +554,8 @@ def get(self):
     'SecondsSinceLastCrash': {
         'data_validation_type': 'enum',
         'default_value': None,
+        'description': '',
         'form_field_choices': None,
-        'form_field_type': None,
```

Comment:
I nixed these. I don't know whether they were used for something in the old days, but they're not used anymore and some of the fields don't have them.

Reply:
I think at some point I inferred the type of form field to use from other values, but never removed this. :+1:

Reply:
That makes a ton of sense, too! Thank you!